adapters/tlsio_wolfssl.c

```diff
@@ -999,6 +1010,12 @@ int tlsio_wolfssl_setoption(CONCRETE_IO_HANDLE tls_io, const char* optionName, c
             tls_io_instance->ignore_host_name_check = *server_name_check;
             result = 0;
         }
+        else if (strcmp("debugLog", optionName) == 0)
+        {
+            (void)wolfSSL_Debugging_ON();
+            (void)wolfSSL_SetLoggingCb(&logging_callback);
```

Comment:
not sure what the return values are here but do we care at all about them?

Reply:
Initially I thought we don't but in the light of this failing it seems like we might.
I also need to add a `LIBWOLFSSL_VERSION_HEX` check and conditionally compile the log routing only if Wolf supports it.